src/components/TableList/TableList.js

```diff
@@ -51,7 +51,9 @@ const generateModelDetailsLink = (modelName, ownerTag, activeUser) => {
 const generateWarningMessage = model => {
   const { messages } = getModelStatusGroupData(model);
   return (
-    <span className="table-list_error-message">{messages.join("; ")}</span>
+    <span className="table-list_error-message" title={messages.join("; ")}>
```

Comment:
instead of joining these twice, store the joined result in a variable and display that?